src/config/backends/power-fan.ts

```diff
@@ -0,0 +1,183 @@
+import { isRight } from 'fp-ts/lib/Either';
+import Reporter from 'io-ts-reporters';
+import * as t from 'io-ts';
+import * as _ from 'lodash';
+
+import { ConfigBackend } from './backend';
+import type { ConfigOptions } from './backend';
+import { schemaTypes } from '../schema-type';
+import log from '../../lib/supervisor-console';
+import * as constants from '../../lib/constants';
+
+const isNullOrUndefined = (v: unknown): v is null | undefined =>
```

Comment:
`x == null` already tests whether x is null or undefined

Reply:
You're right, removed this in https://github.com/balena-os/balena-supervisor/pull/2379/commits/52f3258ffeb7f4dea1d5cb1d3f97bc250bff2bf1